pages/blog/_posts/2020-02-20-Extending-Kubernetes-Static-IP-Routing.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: blogpost
+permalink: /blog/extending-k8s/
+title: "Extending Kubernetes for our needs"
+date: 2020-01-28
+tags: infrastructure kubernetes operator operator-framework networking aws
+author: <a href="https://www.linkedin.com/in/hihrig/">Holger Ihrig</a>
+---
+
+We are using Kubernetes as our cluster scheduler and this serves us well. However we have a 
+few cases where we need to do some additional work.
+
+One case for example is that we have to use static-IPs for some of our services to connect to
+Telecom companies as they expect a single IP address to bind to. This address needs to be the same and
+DNS records are not accepted either. We are running in AWS as well, so the reader might ask why we are not
+using Elastic IPs and add them to the services? Good idea, but Telecom operators will not whitelist
```

Comment:
why we are not using Elastic IPs and add them to the services?
-> why are we not using Elastic IPs and adding them to the services?